### ISSUE TITLE
Will have update ?

### ISSUE THREAD

**Issue by LukasCCB** [eyzi/wumpush#1]
Hello! It is very cool and i want know if it will have continued version. 
Thx for the project.

**Comment by eyzi** [comment]
Yeah, I plan to get it a stable version soon. I just had to set it aside for now so I can work on my game.
